### ISSUE TITLE
Terminal Switcher:  Rename should be reachable with first tab

### ISSUE THREAD

**Issue by kieferrm** [Microsoft/vscode#31835]
While testing #31784.

- VSCode Version: Code - Insiders 1.15.0-insider (f39385b8326671dc6389f21f8c2e2f6afdb352a1, 2017-08-01T05:07:59.267Z)
- OS Version: Darwin x64 16.7.0

---

Steps to Reproduce:

1. Open 3 terminals
2. Select `Terminal: Switch Active Terminal` from the command palette
3. <kbd>ArrowDown</kbd> to the last terminal in the switcher
4. <kbd>Tab</kbd> to rename the terminal
-> keyboard focus disappears from the input box (and seems to be nowhere). I need a second <kbd>Tab</kbd> to move the keyboard focus to the rename action in the selected row. A single <kbd>Tab</kbd> should suffice.

**Comment by Lixire** [comment]
This is also an issue with task configuration
